src/middlewared/middlewared/logger.py

```diff
@@ -17,8 +17,8 @@
 logging.getLogger('aiohttp.internal').setLevel(logging.WARN)
 # We dont need internal botocore debug logging
 logging.getLogger('botocore').setLevel(logging.WARN)
-# we dont need ws4py close debug messages
-logging.getLogger('ws4py').setLevel(logging.WARN)
+# we dont need websocket debug messages
+logging.getLogger('websocket').setLevel(logging.WARN)
```

Comment:
This spams logs way too much for us to merge. We'll just have to bump this until upstream pushes new release to pypy.
```suggestion
logging.getLogger('websocket').setLevel(logging.CRITICAL)
```